Meine finale Version hochgeladen

Sakinashmadi87 opened the pull request.

hussamalafandi committed:
Update deadline for Pull Request submission in README.md

Sakinashmadi87 committed:
Meine finale Version hochgeladen

hussamalafandi commented:
Hallo Sakina,

ich habe deine Abgabe überprüft, und sie erfüllt nicht alle geforderten Punkte. Du musst das Modell trainieren (wenn du eine GPU brauchst, kannst du Colab verwenden). Außerdem musst du deine eigene Trainingsschleife schreiben. Die Trainings-Plots sollen auf Weights & Biases (WandB) hochgeladen werden, und das trainierte Modell muss auf Hugging Face veröffentlicht werden. Bitte lies die Aufgabenstellung noch einmal sorgfältig durch, um Missverständnisse während der Präsentation zu vermeiden.

hussamalafandi commented:
Außerdem: Bitte stelle sicher, dass dein PR nicht auf `master`, sondern auf den Branch mit deinem Namen zeigt.

Sakinashmadi87 commented:
Hi Husam,
https://wandb.ai/ahmadi-sakina-hochschule-hannover/decoder-language-model/runs/mcr5bqb4?nw=nwuserahmadisakina
https://huggingface.co/ahmadisakina/decoder-language-model/tree/main

Am So., 4. Mai 2025 um 12:53 Uhr schrieb Hussam Alafandi <
***@***.***>:

> *hussamalafandi* left a comment (hussamalafandi/GenerativeAI-Project#55)
> <https://github.com/hussamalafandi/GenerativeAI-Project/pull/55#issuecomment-2849142509>
>
> Außerdem: Bitte stelle sicher, dass dein PR nicht auf master, sondern auf
> den Branch mit deinem Namen zeigt.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/hussamalafandi/GenerativeAI-Project/pull/55#issuecomment-2849142509>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMGGBF54H4L6EIBWO2H2TQ324XWUDAVCNFSM6AAAAAB4EEZFFKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDQNBZGE2DENJQHE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

Sakinashmadi87 committed:
Add trainer

Sakinashmadi87 committed:
generate datei  hinzufügen

Sakinashmadi87 committed:
generate advanced hinzufügen

Sakinashmadi87 committed:
dataset hinzufügen

Sakinashmadi87 committed:
upload hinzufügen

Sakinashmadi87 committed:
bonuns aufgabe

Sakinashmadi87 committed:
Bonus (optional)

finetuning ein vortrainiertes Modell wie GPT-2 mithilfe der Hugging Face transformers-Bibliothek

Sakinashmadi87 committed:
Update README.md

README Uptodated
